src/jbi/whiteboard_actions/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .default import *
+from .example import *
```

Comment:
importing a module here enables them to be used within the config JSON. 


https://github.com/mozilla/jira-bugzilla-integration/pull/1/files#r799846714